.github/test_bin/assert_equal

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# USAGE: $ assert_equal STRING1 STRING2
+if [[ "$1" != "$2" ]]; then
+    echo "::error:: Assertion '$1 == $2' failed"
+    exit 1
+fi
```

Comment:
I never got a chance to play with [`bats`](https://github.com/sstephenson/bats), but it would probably be an overhead to add it to this project just for tests I guess

Reply:
Hadn't heard of it, good to know about at least